### ISSUE TITLE
In need of maintenance!

### ISSUE THREAD

**Issue by loleg** [okfn/publicbodies#102]
This code has been untouched for a number of years, and could use some developer love. Even though it is a frontend to a static database, so security warnings are not _especially_ worrying - there are people around the world who may be depending on this content being accessible. Adding SSL certificates, checking up on some overdue issues, and of course updating the underlying datasets is something this community would very much appreciate. 

If you or someone you know can tend to this repository, please drop us a line here: https://discuss.okfn.org/t/winter-maintenance-of-okfn-projects/10116/

**Comment by tfmorris** [comment]
Your profile says that you're a member of OK*, so you should have the privileges to do this yourself. If you need help, @rufuspollock merged the last PR, so might be a good resource for questions.

**Comment by rufuspollock** [comment]
@loleg happy to make you a maintainer here if you don't already have access 😄  - and yes would be great to have some love here ... ❤️

**Comment by loleg** [comment]
Thanks Rufus, but I have quite a lot on my plate as it is, and am just
housekeeping here. To increase the chance of someone stepping in here, I
would suggest adding some:

- Links to relevant community threads / places to contact data maintainers,
e.g. https://github.com/OpendataCH/public_bodies_of_the_swiss_federation
- Analytics of web traffic or other metrics to aim for
- Comments on short/medium/long effort tasks
- Funding opportunities

**Comment by augusto-herrmann** [comment]
For a while I have been meaning to update the Brazilian public bodies dataset and extraction tools (#72). I also would like to advance the data modelling (e.g. #68). This might be a good time to actually do it.

However, I also think that we probably need a new frontend, cleaning/updating other country datasets, finding data maintainers, etc., and I don't have the time to do it all.

**Comment by rufuspollock** [comment]
@augusto-herrmann i think if you update the data we can probably find a way to sort frontend.

**Comment by augusto-herrmann** [comment]
Amazing!

Considering the Open Data Day is coming soon, I think this could be a good ODD project to do.

**Comment by augusto-herrmann** [comment]
I have not managed to do this by the time of the ODD, but I have now proposed including a field (#104) and will soon update the `br.csv` data.

**Comment by augusto-herrmann** [comment]
Done! I've left #104 and #105 up for review for anyone interested and with some time to do it. However, if time goes by and nobody is available, I'll merge them myself.

Maybe this already qualifies for closing this issue. @loleg WDYT?

@rufuspollock we now have links to images (logos or main building photos) of several public bodies. It would be nice to display them on that UI revamp you mentioned.

@loleg do you think it would be a good idea to implement the importing of Swiss data as scripts on this repository (in `scripts/import`) with updates by Github Actions, as I did for Brazilian data in #105?

**Comment by loleg** [comment]
I think that is a great idea, though I don't maintain the morph.io scraper it should be straightforward to port here. Thanks very much for the update!

**Comment by loleg** [comment]
As to closing the issue: feel free to do it. I think though a ticket like this is a good little flag to gain people's attention.

**Comment by augusto-herrmann** [comment]
Of course, @loleg. Thanks for calling attention, as this project had been dormant for a while.

Let me first check the things you mentioned when you opened it.

> Adding SSL certificates, checking up on some overdue issues, and of course updating the underlying datasets is something this community would very much appreciate.

I've just checked, the publicbodies.org has https enabled and a valid TLS certificate just fine. It expires on August 13th, 2021.

As for the overdue issues, I've looked into some, but I still think it would be worth to review all open issues and see if they still make sense today, evaluate if they're worth the effort, if anyone can help, etc., or just simply organising them.

Updating the datasets: it's already been done for BR, which is also configured to automatically update once a week. As time allows for it, we can start updating and automating other data sources as well.

I think anything more specific can be handled on its own issue, so I'm closing this one.
